src/components/ModalSections/index.jsx

```diff
@@ -4,23 +4,80 @@
  */
 
 import React from 'react'
-import { ModalContent } from 'cozy-ui/transpiled/react'
-import Row from 'components/Row'
-import styles from './styles.styl'
+import PropTypes from 'prop-types'
+import { DialogContent } from 'cozy-ui/transpiled/react/Dialog'
+import ListItem from '@material-ui/core/ListItem'
+import ListItemIcon from '@material-ui/core/ListItemIcon'
+import ListItemText from '@material-ui/core/ListItemText'
```

Comment:
We don't need to have Ui's theme on these List components? Should we import theme from UI instead of m-ui?

Reply:
Importing them from material ui is not a problem, the theme is applied to them.

Anyway, it's true that it'd be better to be coherent and import them from cozy-ui. I am doing the changre.